target/linux/ramips/dts/mt7621_humax_e10.dts

```diff
@@ -0,0 +1,153 @@
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mtd/partitions/uimage.h>
+
+/ {
+	compatible = "humax,e10", "mediatek,mt7621-soc";
+	model = "HUMAX QUANTUM E10";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+		label-mac-device = &wan;
+	};
+	
+	gpio-export {
+		compatible = "gpio-export";
+
+		gpio_usb_power {
+                        gpio-export,name = "humax:power:usb";
+                        gpio-export,output = <1>;
+                        gpios = <&gpio 8 GPIO_ACTIVE_HIGH>;
+                };
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "humax:green:power";
+			gpios = <&gpio 6 GPIO_ACTIVE_HIGH>;
+		};
+
+		status {
+			label = "humax:green:status";
```

Comment:
The manual says it's for "pairing"?
https://dearhumax.com/bbs/download.php?bo_table=download&wr_id=17&no=0&wr_1=%EB%B9%A0%EB%A5%B8%20%EC%84%A4%EC%B9%98%20%EA%B0%80%EC%9D%B4%EB%93%9C

Reply:
Corrected!